model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -1081,6 +1081,14 @@ message StandardCoders {
     // Components: the user key coder.
     // Experimental.
     SHARDED_KEY = 15 [(beam_urn) = "beam:coder:sharded_key:v1"];
+
+    //Wraps a coder of a potentially null value
+    //
+    // A Nullable coder encodes nullable values of wrapped coder value that does
+    // not tolerate null values. A Nullable coder uses exactly 1 byte per entry
+    // to indicate whether the value is null, then adds the encoding of the
+    // inner coder for non-null values.
```

Comment:
Could you have this clarify that 1 implies non-null? It would also be nice to update the row coder doc (under "Nullable types in container types (ArrayType, MapType)") to refer to this

Reply:
+1